base.css

```diff
@@ -477,6 +476,10 @@ input[type="checkbox"]::after {
 	background: white;
 	box-shadow: 0 0px 1px rgba(0,0,0,.4), 0 4px 2px rgba(0,0,0,.1);
 	-webkit-transition: background .2s ease-out, left .2s ease-out;
+	-moz-transition: background .2s ease-out, left .2s ease-out;
```

Comment:
is it necessary to use the browser-specific versions still?

MDN say `-webkit-transition` is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-webkit-transition

Reply:
@benmccann thank you for reviewing this. 

Originally site-kit has `-webkit-transition` and visual studio code was reporting lint warnings and I went ahead and added all vendor prefix. 

PS.  just adding `transition` stops lint warnings too.

Reply:
Also I would be happy to revert them just how it was before

Reply:
My question is can you put just `transition` and remove the others?

Reply:
I just added the `transition`